assets/src/edit-story/components/library/panes/shapes/shapesPane.js

```diff
@@ -63,11 +63,6 @@ function ShapesPane(props) {
       <Header>
         <Title>{__('Shapes', 'web-stories')}</Title>
       </Header>
-      <SearchInput
```

Comment:
Seems like both shapes and text panel supposed to support search input per figma?

Reply:
Correct, but probably not now (currently non-functional, so removed) https://github.com/google/web-stories-wp/issues/520#issuecomment-598379439

Reply:
Ok. Let's just make sure we file a bug if removing an expected feature.